src/font/lv_font.h

```diff
@@ -102,29 +102,29 @@ typedef _lv_font_kerning_t lv_font_kerning_t;
 typedef uint8_t lv_font_kerning_t;
 #endif /*DOXYGEN*/
 
+typedef bool(*lv_font_glyph_get_info_cb_t)
+(const lv_font_t * font, lv_font_glyph_dsc_t * g_dsc, uint32_t unicode, uint32_t unicode_next);
+typedef const void * (*lv_font_glyph_acquire_draw_data_cb_t)(lv_font_glyph_dsc_t * g_dsc, lv_draw_buf_t * draw_buf);
+typedef void (*lv_font_glyph_release_draw_data_cb_t)(lv_font_glyph_dsc_t * g_dsc);
+
 /** Describe the properties of a font*/
 struct _lv_font_t {
-    /** Get a glyph's descriptor from a font*/
-    bool (*get_glyph_dsc)(const lv_font_t *, lv_font_glyph_dsc_t *, uint32_t letter, uint32_t letter_next);
-
-    /** Get a glyph's bitmap from a font*/
-    const void * (*get_glyph_bitmap)(lv_font_glyph_dsc_t *, lv_draw_buf_t *);
-
-    /** Release a glyph*/
-    void (*release_glyph)(const lv_font_t *, lv_font_glyph_dsc_t *);
+    lv_font_glyph_get_info_cb_t          get_glyph_dsc;            /**< Get a glyph's descriptor from a font*/
+    lv_font_glyph_acquire_draw_data_cb_t get_glyph_bitmap;         /**< Acquire a glyph's bitmap from a font*/
+    lv_font_glyph_release_draw_data_cb_t glyph_release_draw_data;  /**< Release a glyph*/
```

Comment:
```suggestion
    lv_font_glyph_release_draw_data_cb_t release_glyph_draw_data;  /**< Release a glyph*/
```

Reply:
Because I think glyph is a sub-part of the font, I used `lv_font_glyph_release_draw_data_cb_t`, Therefore, glpyh's previous writing is also adopted on the callback function. I'm not sure which naming method is better, but I think it is doomed, so I chose lv_font_glyph_release_draw_data_cb_t.

In the future, when the major version will be updated. I want to replace the API with
```cpp
lv_font_glyph_get_info_cb_t          glyph_get_info;            /**< Get a glyph's descriptor from a font*/
lv_font_glyph_acquire_draw_data_cb_t glyph_acquire_draw_data;   /**< Acquire a glyph's bitmap from a font*/
lv_font_glyph_release_draw_data_cb_t glyph_release_draw_data    /**< Release a glyph*/
```

Of course, another way to write it is,
```cpp
lv_font_get_glyph_info_cb_t          get_glyph_info;            /**< Get a glyph's descriptor from a font*/
lv_font_acquire_glyph_draw_data_cb_t acquire_glyph_draw_data;   /**< Acquire a glyph's bitmap from a font*/
lv_font_release_glyph_draw_data_cb_t release_glyph_draw_data    /**< Release a glyph*/
```

I think.

Reply:
I think both are fine, just be consistent:
1. `<action>` (get/release) + `glyph` + <subject> (info/draw_data)
2. `glyph` + `<action>` (get/release) + <subject> (info/draw_data)

In an object oriented language we would use it like: 
```c
//1
font1.get_glyph_bitmap(gdsc, draw_buf);

//2
font1.glyph_get_bitmap(gdsc, draw_buf);
```

For me starting with the `action` sounds better, but it's subjective I guess.

Once talking about OOP, I've just noticed that these callbacks should have the font as their first parameter to follow [these rules](https://docs.lvgl.io/master/CODING_STYLE.html#api-conventions):
> To register and use callbacks one of the following needs to be followed.
>  - Pass a pointer to a struct as the first argument of both the registration function and the callback. That struct must contain void * user_data field.

Reply:
> I think both are fine, just be consistent:
> 
> 1. `<action>` (get/release) + `glyph` +  (info/draw_data)

Ok, I think I will take this way now, because another way will cause more changes.

> Once talking about OOP, I've just noticed that these callbacks should have the font as their first parameter to follow [these rules](https://docs.lvgl.io/master/CODING_STYLE.html#api-conventions):
> 
> > To register and use callbacks one of the following needs to be followed.
> > 
> > * Pass a pointer to a struct as the first argument of both the registration function and the callback. That struct must contain void * user_data field.

Sure, but in fact, these two (`get_glyph_bitmap `/`release_glyph_bitmap`) `callbacks` operate on `glyph_dsc`. They don't need a `font` object.

Reply:
> Sure, but in fact, these two (get_glyph_bitmap /release_glyph_bitmap) callbacks operate on glyph_dsc. They don't need a font object.

However, they are part of the `lv_font_t` structure and the MicroPythin binding stores some data in `user_data` to handle the callbacks correctly. Therefore the callbacks first parameter needs to be a pointer to their `struct` where MicroPython can find the  `user_data`.